src/utils/validateUserSlippageTolerance.ts

```diff
@@ -0,0 +1,21 @@
+import { Percent } from '@uniswap/sdk-core'
+
+export enum SlippageValidationResult {
+  TooLow = 'TOO_LOW',
+  TooHigh = 'TOO_HIGH',
+  Valid = 'VALID',
```

Comment:
nit: I thought we named enums in all caps, eg `TOO_LOW`.

Reply:
I am not aware of that styling convention, searching the codebase we have a mix of both. We may want to standardise in the future.